app/models/moment.rb

```diff
@@ -57,6 +59,13 @@ def self.find_secret_share!(identifier)
     )
   end
 
+  def self.populate_moments_moods
+    Moment.all.find_each do |moment|
+      moment.mood = Mood.where(id: moment.mood).pluck(:id)
```

Comment:
Can the `where` be replaced with `find`?

Reply:
no because there may be many moods per moment. `find` will only get one.
This is a case that will look like `moment.mood = Mood.where(id: [1,2,3]).pluck(:id)` and create many Moods for one Moment.

Reply:
Ah yes! LGTM then!